src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/renametool/SpecialRenameUserPage.java

```diff
@@ -57,6 +61,12 @@ public SpecialRenameUserPage fillFormData(String newUsername, String confirmUser
   }
 
   public SpecialRenameUserPage submitChange() {
+    try {
+      Thread.sleep(10000);
+    }catch (Exception e){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00108)

Reply:
again